src/Types/CborBytes.purs

```diff
@@ -0,0 +1,79 @@
+-- | CborBytes. A wrapper over `ByteArray` to indicate that the bytes are cbor.
+
+module Types.CborBytes
+  ( CborBytes(..)
+  , cborBytesToByteArray
+  , cborBytesFromByteArray
+  , cborBytesToIntArray
+  , cborBytesFromIntArray
+  , cborBytesFromIntArrayUnsafe
+  , cborBytesToHex
+  , byteLength
+  , hexToCborBytes
+  , hexToCborBytesUnsafe
+  ) where
+
+import Data.Newtype (class Newtype, wrap, unwrap)
+
+import Types.ByteArray (ByteArray)
+import Types.ByteArray as BA
```

Comment:
I think we've started avoiding one- or two-letter qualified module names. `ByteArray` would be better I think